### PR TITLE
[IDP-1768] Add experimental MSW plugin

### PR DESCRIPTION
**Pull request by tjosepo**
## Description of changes

Adds a small plugin that adds the boilerplate required for `openapi-msw`. Not sure how useful that is, but eh. It's there.

Marked the plugin as "experimental" in case we ever want to change it in the future. (Not sure if necessary, or if it's implied with the package being in 0.x.x)

## Breaking changes
N/A
## Additional checks

- [x] Updated the documentation of the project to reflect the changes
- [x] Added new tests that cover the code changes

### CODE REVIEW

File: debug/src/codegen/v1/client.ts [hunk 1]
```diff
@@ -1,5 +1,4 @@
-/** Do not modify. This file has been generated by @workleap/create-schemas */
+/** This file has been generated by @workleap/create-schemas (https://github.com/gsoft-inc/wl-openapi-typescript). Do not modify manually. */
```
Comment (tjosepo) [inline debug/src/codegen/v1/client.ts:1]: Unrelated to the current change, but this file had not been generated since the last PR, so this comment has only been updated now.

File: packages/create-schemas/tests/plugins.test.ts [hunk 2]
```diff
@@ -69,4 +70,33 @@ describe.concurrent("plugins", () => {
           "
         `);
     });
+
+    test("openapiMSWPlugin", async ({ expect }) => {
```
Comment (mahmoudmoravej) [inline packages/create-schemas/tests/plugins.test.ts:74]: Question: Do we have any guideline for how we write test cases? It is usually a good practice to write test scenarios in a sentence.(e.g. [an example](https://github.com/Shopify/web-configs/blob/main/packages/babel-preset/tests/babel-preset.test.ts#L24)) For example: `openapiMSWPlugin generates right files when it is configured`.

File: packages/create-schemas/tests/plugins.test.ts [hunk 3]
```diff
@@ -69,4 +70,33 @@ describe.concurrent("plugins", () => {
...
+        assert(emittedFile);
+
+        expect(emittedFile.filename).toBe("openapi-msw.ts");
+        expect(emittedFile.code).toMatchInlineSnapshot(`
+          "import type { paths } from "./types.ts";
+          import { createOpenApiHttp } from "openapi-msw";
+
+          export const http = createOpenApiHttp<paths>();"
+        `);
+    });
```
Comment (mahmoudmoravej) [inline packages/create-schemas/tests/plugins.test.ts:101]: [Suggestion] : Good to have test for the main edge case: 
`should not call transform function when id does not match openapiTypeScriptId`

**Review by mahmoudmoravej — APPROVED** [review]
🎩 & LGTM ✅ 🎉 
I am ok to keep the `unstable_`  as it is experimental even we don't have a public release yet. We need to have a real case to test and confirm the usage before offering it as main functionality.

**Comment by patricklafrance** [comment]
> 🎩 & LGTM ✅ 🎉 I am ok to keep the `unstable_` as it is experimental even we don't have a public release yet. We need to have a real case to test and confirm the usage before offering it as main functionality.

This comment makes me thought about this... It's kind of the same situation for the "experimental" http client, should the http client plugin also be prefixed with "unstable_" @tjosepo ?

File: packages/create-schemas/tests/plugins.test.ts (hunk 2 above)
Comment (tjosepo) [reply packages/create-schemas/tests/plugins.test.ts:74]: We don't have a convention for test names. So far, I've favored shorter names, but I don't mind using long names, especially as we add more test cases.

**Review by heqianwang — APPROVED** [review]
